app/pages/admin/project-status.jsx

```diff
@@ -85,10 +86,11 @@ class ProjectStatus extends Component {
     this.setState({ dialogIsOpen: false });
   }
 
-  handleDialogSuccess(workflow) {
+  handleDialogSuccess() {
+    const defaultWorkflow = this.state.workflows.filter(workflow => workflow.id === this.state.defaultWorkflowId);
```

Comment:
I wanted to call this variable `workflow`, but that raised a linter error (`'workflow' is already declared in the upper scope`), so I went with defaultWorkflow.